OctoKit/Issue.swift

```diff
@@ -3,34 +3,80 @@ import RequestKit
 
 // MARK: model
 
+public enum IssueState: String {
```

Comment:
As this will be used for the Milestone as well, it should have different naming. `State` is not really an alternative.